src/scripts/admin/admin.actions.js

```diff
@@ -10,7 +10,9 @@ var Actions = Reflux.createActions([
     'removeUser',
     'toggleSuperUser',
     'unBlacklistUser',
-    'update'
+    'update',
+    'searchUsername',
+    'toggleAdmin'
```

Comment:
I would change 'toggleAdmin' to something like 'filterAdmin' so it's not confused with 'toggleSuperUser'